lib/simulator-xcode-6.js

```diff
@@ -772,6 +772,15 @@ class SimulatorXcode6 extends EventEmitter {
     await settings.setReduceMotion(this, reduceMotion);
   }
 
+  /**
+   * Enable/Disable reduce transparency.
+   *
+   * @param {boolean} reduceMotion - Whether or not to enable it.
+   */
+  async setReduceTransparency (reduceTransparency = true) {
```

Comment:
it also makes sense to remove the default value there

Reply:
fixed https://github.com/appium/appium-ios-simulator/pull/342/commits/4140047b29e3772e22e2d67032b9e8812b54aec5